lib/components/Stepper/types.ts

```diff
@@ -1,11 +1,11 @@
 import { TypographyProps } from "@components/Typography/base";
 
 export type TBasicStepperProps = {
-    currentStep: Steps;
+    currentStep: Steps.StepOne | Steps.StepTwo | Steps.StepThree;
```

Comment:
Do we only have 3 steps? Can this be dynamic, I think we can have more steps than this

Reply:
thank you🙇
improved✅